README.md

```diff
@@ -33,7 +34,7 @@ Tauri Apps
 ## Introduction
 Tauri is a framework for building tiny, blazing fast binaries for all major desktop platforms. Developers can integrate any front-end framework that compiles to HTML, JS and CSS for building their user interface. The backend of the application is a rust-sourced binary with an API that the front-end can interact with.
 
-The user interface in Tauri apps currently leverages Cocoa/WebKit on macOS, gtk-webkit2 on Linux and MSHTML (IE10/11) or Webkit via Edge on Windows. **Tauri** uses (and contributes to) the MIT licensed project that you can find at [webview](https://github.com/webview/webview) and the [related webview organization](https://github.com/webview).
+The user interface in Tauri apps currently leverages [`winit`](https://docs.rs/winit) as a WebView abstraction for all platforms via the **Tauri-team** incubated and maintained [wry](https://github.com/tauri-apps/wry), which creates a unified interface to these system webviews.
```

Comment:
winit is a window lib, maybe we should note that we use gtk on linux and still mention wkwebview/webview2/webkitgtk